projects/angular/src/forms/common/label.ts

```diff
@@ -64,14 +70,9 @@ export class ClrLabel implements OnInit, OnDestroy {
     ) {
       this.renderer.addClass(this.el.nativeElement, 'clr-col-12');
       this.renderer.addClass(this.el.nativeElement, `clr-col-md-${this.layoutService.labelSize}`);
-    }
-    if (this.controlIdService && !this.forAttr) {
-      this.subscriptions.push(
-        this.controlIdService.idChange.subscribe(id => {
-          this.forAttr = id;
-          this.idAttr = this.idInput || `${id}-label`;
-        })
-      );
+      if (this.controlIdService && !this.forAttr) {
+        this.subscriptions.push(this.controlIdService.idChange.subscribe(id => (this.forAttr = id)));
```

Comment:
```suggestion
        this.subscriptions.push(this.controlIdService.idChange.subscribe(id => {
          this.forAttr = id;
        }));
```

A statement block should have braces.